src/ui/public/jquery/__tests__/findTestSubject.js

```diff
@@ -0,0 +1,59 @@
+var $ = require('jquery');
```

Comment:
I'd like to see a test where a simple (not compound) selector returns multiple elements in the result.